plugin/trino-memory/src/test/java/io/trino/plugin/memory/TestMemoryConnectorTest.java

```diff
@@ -556,7 +556,7 @@ public void testViews()
         assertUpdate("CREATE OR REPLACE VIEW test_view AS " + query);
 
         assertQueryFails("CREATE TABLE test_view (x date)", "View \\[default.test_view] already exists");
-        assertQueryFails("CREATE VIEW test_view AS SELECT 123 x", "View already exists: default.test_view");
+        assertQueryFails("CREATE VIEW test_view AS SELECT 123 x", ".*View already exists: 'memory.default.test_view'");
```

Comment:
Do we need `.*` as a prefix?

Reply:
Yes, SemanticException will extractLocation like ``line 1:1: View already exists: 'memory.default.test_view'``